.github/workflows/auto-version-bump-on-staging.yml

```
@@ -0,0 +1,75 @@
+          git commit -m "chore(release): Bump version to ${NEW_VERSION} after main merge"
+
+      - name: Push changes to staging
+        run: git push origin staging
+
+      - name: Create Pull Request to main
+        uses: peter-evans/create-pull-request@v7
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          title: "Release ${{ steps.get_version.outputs.NEW_VERSION }}"
+          body: "Automated release pull request after merging to main."
+          head: staging
```

actionlint (1.7.8)

input "head" is not defined in action "peter-evans/create-pull-request@v7". available inputs are "add-paths", "assignees", "author", "base", "body", "body-path", "branch", "branch-suffix", "branch-token", "commit-message", "committer", "delete-branch", "draft", "labels", "maintainer-can-modify", "milestone", "path", "push-to-fork", "reviewers", "sign-commits", "signoff", "team-reviewers", "title", "token"

(action)